test/direct-message-key.test.js

```diff
@@ -37,19 +38,34 @@ test('direct-message-key', t => {
   t.deepEqual(
     directMessageKey.easy(mySSBKeys)(yourSSBKeys.id),
     key1,
-    'DirectMessageKey.easy produces same result (classic/classic)'
+    'DirectMessageKey.easy produces same result (classic keys + sigil)'
   )
 
+  const yourEd25519URI = fromFeedSigil(yourSSBKeys.id) // ssb:feed/ed25519
+  t.deepEqual(
+    directMessageKey.easy(mySSBKeys)(yourEd25519URI),
+    key1,
+    'DirectMessageKey.easy produces same result (classic keys + ssb:feed/ed25519)'
+  )
+
+  // const yourClassicURI = yourEd25519URI.replace('ed25519', 'classic') // ssb:feed/classic
+  // t.deepEqual(
+  //   directMessageKey.easy(mySSBKeys)(yourClassicURI),
+  //   key1,
+  //   'DirectMessageKey.easy produces same result (classic keys + ssb:feed/classic)'
+  // )
+
+  console.log(bendyKeys.id)
   t.deepEqual(
     directMessageKey.easy(mySSBKeys)(bendyKeys.id),
     key3,
-    'DirectMessageKey.easy produces same result (classic/bendy butt)'
+    'DirectMessageKey.easy produces same result (classic keys + ssb:feed/bendybutt-v1)'
```

Comment:
> Ok, how does bendybutt SSB URIs behave with this module? Because I think we strictly need to support that but we don't strictly need to support ssb:feed/ed25519 although it's nice.

cc @staltz it passes fine

Reply:
These URI's are currently supported:

```js
// from ssb-uri2/lib.js

        if (parts.format !== 'ed25519' &&
            parts.format !== 'bendybutt-v1' &&
            parts.format !== 'gabbygrove-v1' &&
            parts.format !== 'buttwoo-v1') {
            throw new Error('Unknown format for type "feed": ' + parts.format);
        }
```